src/autoload.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/* An autoloader for ReCaptcha\Foo classes. This should be require()d
+ * by the user before attempting to instantiate any of the ReCaptcha
+ * classes.
+ */
+
+spl_autoload_register(function ($class) {
+    /* All of the classes have names like "ReCaptcha\Foo", so we need
```

Comment:
Restrict this to **only** autoload the `ReCaptcha` namespace. So, just early out of the function for anything else.